tests/features/common/kie-controller-common.feature

```diff
@@ -1,7 +1,15 @@
 @rhdm-7/rhdm-controller-rhel8 @rhpam-7/rhpam-controller-rhel8
 Feature: KIE Controller configuration common tests
 
-    # https://issues.jboss.org/browse/RHPAM-891
+  Scenario: Test REST API is secure
+    When container is ready
+    Then check that page is served
+      | property             | value               |
+      | port                 | 8080                |
+      | path                 | /management/servers |
+      | expected_status_code | 403                 |
```

Comment:
If I'm not mistaken, this test is moved from [here](https://github.com/jboss-container-images/rhdm-7-image/pull/380/files#diff-775273de5a8c2b0a985788874ff774f61cb0b70fb1d324e4262a6b5a6537783aL10). Just a question why is this return code changed from the test?

Reply:
these tests were never run or at least there has beed a long time since it was test by someone.
it was not working with 201.